.github/workflows/publish-image-and-helm-on-tag.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Publish image to GHCR and Helm chart to GH pages on tag'
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+concurrency:
+  group: spsh-loadtest-main
+  cancel-in-progress: true
+
+jobs:
+  publish_docker:
+    permissions:
+      packages: write
+      security-events: write
+      contents: read
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/image-publish-trivy.yaml@5.0.0
+    with:
+      image_name: "spsh-k6-loadtest"
```

Comment:
i would also like the image name to change to the repo name: schulportal-load-tests